EXAMPLES.md

```diff
@@ -319,11 +321,18 @@ ShopifyBuy.Client().products((products, error) => {
 
     cart.LineItems.AddOrUpdate(productVariantToCheckout, 1);
 
+    var key = null;
+    #if UNITY_IOS
+    key = "com.merchant.id";
+    #elif UNITY_ANDROID
+    key = "BL9QiRljozDhgfyfVHoK+l1l98fBY0x/in0rCYJxmTfnzJDWsX1+8l4HEa4LO0WeKQlYtuk8zcJtzimTMhr1UL8=";
```

Comment:
Is this the same for all merchants, or will it have to be customized?

Reply:
@richardmonette each merchant has its own, which is provided by the admin page. Instructions mention that. These two strings are placeholders for the sake of the example. Game devs will not use `com.merchant.id` for iOS either.